single-node/Vagrantfile

```diff
@@ -25,7 +25,7 @@ Vagrant.configure("2") do |config|
   config.ssh.insert_key = false
 
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 962.0.0"
+  config.vm.box_version = ">= 1109.1.0"
```

Comment:
People could still want to change the channel to stable, and this version would currently be too high. Also, elsewhere in this PR this is set at v1122.0.0